contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -973,53 +973,52 @@ contract NodeOperatorsRegistry is AragonApp, IStakingModule, Versioned {
         return TYPE_POSITION.getStorageBytes32();
     }
 
-    /// @notice Returns the validators stats of all node operators in the staking module
-    /// @return exitedValidatorsCount Total number of validators in the EXITED state
-    /// @return activeValidatorsKeysCount Total number of validators in active state
-    /// @return readyToDepositValidatorsKeysCount Total number of validators ready to be deposited
-    function getValidatorsKeysStats()
-        external
-        view
-        returns (
-            uint256 exitedValidatorsCount,
-            uint256 activeValidatorsKeysCount,
-            uint256 readyToDepositValidatorsKeysCount
-        )
-    {
+    // struct ValidatorsReport {
+    //     uint256 totalExited;
+    //     uint256 totalDeposited;
+    //     uint256 totalVetted;
```

Comment:
I see an inconsistency in naming `totalVetted` vs `targetLimit`. maybe `currentLimit` and `targetLimit`?

Reply:
Such naming was introduced in the #557 PR and was preserved for consistency.